Bug fix in xml2stm.py

yshalabi opened the pull request:
Some manually transcribed words have no values. For example see word 1829
from file vim dev_xml_2016_02_17/bw/5331FFF2-358E-445E-BA71-74B745387304.xml
(line number 2114).
These will result in a python error due to indexing an empty childNodes list.

yshalabi committed:
Some manually transcribed words have no values. For example see word 1829
from file vim dev_xml_2016_02_17/bw/5331FFF2-358E-445E-BA71-74B745387304.xml
(line number 2114).
These will result in a python error due to indexing an empty
childNodes list.

sameerkhurana10 commented:
Yes, you are right. There are two ways to fix the issue.

1. What you suggested
2. Handle this while generating the dev transcript xmls

If we handle this in xml generation, we will have to generate new dev transcript xmls and upload to the ftp server.

@yifan and @amali what do you guys suggest?

yshalabi committed:
Update mgb_data_prep.sh

Correctly deleting stale reco2file_channel. In fact the append operation is unnecessary, but keeping it to stay consistent with other code that also follows this pattern (for whatever reason...)

yshalabi committed:
Merge pull request #1 from yshalabi/reco2file_channel-bug

Update mgb_data_prep.sh
